CHANGELOG.rst

```diff
@@ -2,6 +2,23 @@
 Wijzigingen
 ===========
 
+1.1.0 (2022-05-17)
+===========
+
+API spec changes
+----------------
+
+* Add authorisaties documentation
+* Add notificaties documentation
+
+
+Implementation changes
+----------------
+
+* Use github actions CI
+* Fix requesting non-existent file (#584)
+* Allow uploading large files in chunks
+
 1.0.2 (...)
```

Comment:
@sergei-maertens kunnen deze wijzigingen meegenomen worden naar `1.1.0`? Volgens mij is deze release destijds ook niet gebeurd.

Reply:
Sowieso is het niet correct dat die `ETag` spullen onder 1.0.x staan - dat is echt een 1.1.0 feature, maar dat hele lijstje lijkt me gewoon changelog voor 1.1.0.

Wat betreft de bugfixes:

> * Remove audittrail logging and notifications from ``ObjectInformatieObject`` resource
> * Fix documentation references in API schema description (`Issue 1552`_)

-> dit lijkt impact te hebben op API schema, dus ik zou even kortsluiten met Michiel of dit ook nog in een 1.0.2 gereleased moet worden, of dat de support voor 1.0.x voorbij is.